client/src/reducers/datasetMetadata.ts

```diff
@@ -1,24 +1,24 @@
 /*
- Dataset metadata reducer, modifies Portal collections-related state. 
+ Dataset metadata reducer, modifies Portal collections-related state.
  */
 
 // Core dependencies
-import { Action } from "redux";
+import { Action, AnyAction } from "redux";
 
 // App dependencies
 import { DatasetMetadata as IDatasetMetadata } from "../common/types/entities";
 
 /*
  Action dispatched on successful response from dataset-metadata endpoint.
  */
-export interface DatasetMetdataAction extends Action<string> {
+export interface DatasetMetadataAction extends Action<string> {
```

Comment:
I use https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker to catch typos 🤖 ✅ !